src/app/services/services.module.ts

```diff
@@ -84,6 +84,7 @@ import { TokenService as TokenServiceAbstraction } from 'jslib/abstractions/toke
 import { TotpService as TotpServiceAbstraction } from 'jslib/abstractions/totp.service';
 import { UserService as UserServiceAbstraction } from 'jslib/abstractions/user.service';
 import { VaultTimeoutService as VaultTimeoutServiceAbstraction } from 'jslib/abstractions/vaultTimeout.service';
+import { ConsoleLogService } from 'jslib/services/consoleLog.service';
```

Comment:
Please move this import up into the `jslib/services/**` block above with the other service implementation imports.